_posts/2019-04-03-cockpit-191.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Cockpit 191
+author: mvo
+date: 2019-04-03
+tags: cockpit linux
+slug: cockpit-191
+category: release
+summary: Cockpit with Machines and Networking improvements
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.  Here are the release notes from version 191.
+
+### Machines: iSCSI Storage pools
+
+You can now add a iSCSI target as a storage pool for your virtual
+machines.
+
+![iSCSI storage pool](/images/machines-iscsi.png)
+
+### Machines: Better notifications
+
+Notifications about virtual machines have been streamlined and use a
+common presentation now.
+
+![Notifications](/images/machines-notifications.png)
+
+### Network: Ports in the Firewall
+
+In addition to services, which represent a predefined set of ports,
+you now can now open individual ports in the firewall by directly
+specifying their number.
+
+![Firewall ports](/images/networking-ports.png)
+
+### System: CPU security mitigation
+
+Cockpit can now help you with enabling certain security vulnerability mitigations.
```

Comment:
perhaps: "..., starting with the [nosmt option](https://access.redhat.com/security/vulnerabilities/L1TF)." ?

Reply:
Yes, good idea. Fixed.